models/src/main/protobuf/CasperMessage.proto

```diff
@@ -183,6 +183,7 @@ message BodyProto {
 
 message RejectedDeployProto{
   bytes sig = 1;
+  bool  unRejected = 2;
```

Comment:
This object now looks a little bit strange.

Maybe we can rename the flag to be just `rejected` and the whole object something like `ConflictingDeploy` or with merging in name like `MergingDeployStatus`?